ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_number_of_dimensions.py

```diff
@@ -0,0 +1,61 @@
+import ivy.functional.backends.numpy as ivy_np
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+import numpy as np
+from hypothesis import given
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# squeeze
+@st.composite
+def _squeeze_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(), key="value_shape"))
+    valid_axes = []
+    for index, axis in enumerate(shape):
+        if axis == 1:
+            valid_axes.append(index)
+    valid_axes.insert(0, None)
+
+    return draw(st.sampled_from(valid_axes))
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_dtypes,
+        shape=st.shared(helpers.get_shape(), key="value_shape"),
+    ),
+    axis=_squeeze_helper(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.squeeze"
+    ),
+    as_variable=helpers.array_bools(),
+    with_out=st.booleans(),
+    native_array=helpers.array_bools(),
+)
+def test_numpy_squeeze(
+    dtype_and_x,
+    axis,
+    num_positional_args,
+    native_array,
+    as_variable,
+    with_out,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    input_dtype = [input_dtype]
+    np_frontend_helpers.test_frontend_function(
```

Comment:
I've found out the `where` arg error issue, please use `helper.test_frontend_function` instead because the `numpy_frontend` version is for function requiring `where` testing, this should fix the issue, but let me know if there is further errors!